src/main/java/uk/gov/hmcts/reform/prl/handlers/CaseEventHandler.java

```diff
@@ -5,13 +5,13 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.prl.clients.ccd.CoreCaseDataService;
```

Comment:
You just copied the existing event from one file to another. It is not serving either the purpose or reflecting the changes intended.

Reply:
@AlokDatta  The triggerEvent function in CoreCaseData Service is being passed a map containing additional values like taskList and respondentTaskList when called from the CaseEventHandler. These are not available in CaseData. Hence I wasn't sure if we could replace this with the CaseData from StartEventResponse.